BackEndFlask/Functions/teamImport.py

```diff
@@ -41,12 +39,12 @@ def teamcsvToDB(teamFile, owner_id, course_id):
         teamFile = xlsx_to_csv(teamFile)
     try:
         with open(teamFile, mode='r', encoding='utf-8-sig') as teamcsv:
-            team_courses = get_team_courses_by_course_id(course_id)
-            if type(team_courses) is type(""):
+            teams = get_team_by_course_id(course_id)
+            if type(teams) is type(""):
                 delete_xlsx(teamFile, isXlsx)
-                return team_courses
-            for team_course in team_courses:
-                deactivated_team = deactivate_team(team_course.team_id)
+                return teams
+            for team in teams:
+                deactivated_team = deactivate_team(team.team_id)
```

Comment:
I don't see where deactivated_team gets used other than checking if it error'd out, so maybe name this to error_msg or something similar.